geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -675,6 +679,47 @@ Occurred on following members
 1. ubuntu(server1:17682)<v1>:27574
 ```
 
+## <a id="create_lucene_index" class="no-quick-link"></a>create lucene index
+
+Create a Lucene index.
+
+See also [describe lucene index](describe.html#describe_lucene_index), [destroy lucene index](destroy.html#destroy_lucene_index), [list lucene indexes](list.html#list_lucene_indexes) and [search lucene](search.html#search_lucene).
+
+**Availability:** Online. You must be connected in <span class="keyword parmname">gfsh</span> to a JMX Manager member to use this command.
+
+**Syntax:**
+
+``` pre
+create lucene index --name=value --region=value --field=value(,value)* [--analyzer=value(,value)*] [--group=value(,value)*]
+```
+
+| Name                                               | Description                                                                            | Default |
+|----------------------------------------------------|----------------------------------------------------------------------------------------|---------|
+| <span class="keyword parmname">\\-\\-name</span>       | *Required.* Name of the index to create.                                               |         |
+| <span class="keyword parmname">\\-\\-region</span>     | *Required.* Name/Path of the region which corresponds to the "from" clause in a query. |         |
+| <span class="keyword parmname">\\-\\-field</span>      | *Required.* Field of the region values that are referenced by the index.               |         |
+| <span class="keyword parmname">\\-\\-analyzer</span>   | Analyzer to extract terms from text                                  |         |
+| <span class="keyword parmname">\\-\\-group</span>      | The index will be created on all the members in the specified member groups.                     |         |
+
+<span class="tablecap">Table 6. Create Index Parameters</span>
```

Comment:
Table labeling comes out wrong.  There are 2 tables labelled with number 6, and table 7 has 2 labels.  Maybe just remove table labels completely?